compiler/renderer.nim

```diff
@@ -820,14 +820,24 @@ proc gTypeClassTy(g: var TSrcGen, n: PNode) =
   dedent(g)
 
 proc gblock(g: var TSrcGen, n: PNode) =
+  if n.len == 0:
```

Comment:
can you simply write:
```nim
  if n.len != 2: return # block stmt should have two children
```
and remove the rest of the diff?

Reply:
It's intended for better error messages.

Reply:
ok; then maybe add a short comment to explain this so that the next person doesn't do this "wrong simplification" ?

Reply:
sure